### PR TITLE
Setup staff #26

### LINKED ISSUE #26: Setup basic data pipeline

### PR DESCRIPTION
**Pull request by Ssstlis**
Close #26 

I've just setup some stuff for actually run server with applying migrations

### CODE REVIEW

File: new/src/main/resources/application.conf [hunk 1]
```diff
@@ -14,5 +14,11 @@ app {
     password = ${?POSTGRES_PASSWORD}
     threadPoolSize = 32
     chunks = 50
+    maxLifetime = ${?POSTGRES_LIFETIME}
+    connTimeout = ${?POSTGRES_CONN_TIMEOUT}
+    locations = [
+      "migration/prod"
+//      ,"migration/dev"
```
Comment (blessedvictim) [inline new/src/main/resources/application.conf:21]: wtf comment?!

**Comment by Ssstlis** [comment]
Full updated codecov report is [here](https://codecov.io/gh/weakteam/vicly_back/pull/27/tree/new/src/main/scala/io/github/weakteam)

**Comment by Ssstlis** [comment]
I'm sorry, a forget all of important staff, please stay tuned for further updates! :(

File: new/src/main/resources/application.conf (hunk 1 above)
Comment (Ssstlis) [reply new/src/main/resources/application.conf:21]: By now we might haven't dev migration in production config. I'll fix configs later.

**Pull request merged**